server/static/static.go

```diff
@@ -39,6 +39,7 @@ var (
 	expandedSuggestionsEnabled = flag.Bool("app.expanded_suggestions_enabled", false, "If set, enable more build suggestions in the UI.")
 	enableWorkflows            = flag.Bool("remote_execution.enable_workflows", false, "Whether to enable BuildBuddy workflows.")
 	enableExecutorKeyCreation  = flag.Bool("remote_execution.enable_executor_key_creation", false, "If enabled, UI will allow executor keys to be created.")
+	testOutputManifestsEnabled = flag.Bool("app.test_output_manifests_enabled", false, "If set, the target page will render the contents of test output zips.")
```

Comment:
you could default this to true if you want

Reply:
sure yeah done